packages/amplify_datastore/example/ios/amplify_datastore_unit_tests/DataStorePluginUnitTests.swift

```diff
@@ -334,4 +334,51 @@ class DataStorePluginUnitTests: XCTestCase {
         // Make sure that the event was indeed sent successfully
         wait(for: [eventSentExp!], timeout: 1)
     }
+    
+    func test_clear_success() throws {
+    
+    class MockDataStoreBridge: DataStoreBridge {
+        override func onClear(
+            completion: @escaping DataStoreCallback<Void>) throws {
+            // Return from the mock
+            completion(.emptyResult)
+        }
+    }
+        let dataStoreBridge: MockDataStoreBridge = MockDataStoreBridge()
+        pluginUnderTest = SwiftAmplifyDataStorePlugin(bridge: dataStoreBridge, flutterModelRegistration: flutterModelSchemaRegistration)
+        
+        pluginUnderTest.onClear(
+            flutterResult: {(result) in
+                XCTAssertNil(result)
+            })
+    }
```

Comment:
Indentation seems off. You can run the Xcode's default formatter for indentation. We will later invest in swiftlint etc.